synfig-studio/src/gui/canvasview.cpp

```diff
@@ -1997,6 +1997,7 @@ CanvasView::create_tab_label()
 	label->show();
 
 	closebutton = manage(new Gtk::Button());
+	closebutton->set_margin_right(4);
```

Comment:
`set_margin_right()` is [deprecated](https://developer.gnome.org/gtkmm/stable/classGtk_1_1Widget.html#ac9e58b824148641fea1fa41bfe257efb)... use
`set_margin_end()` instead

Reply:
Resolved...